tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -2777,6 +2777,70 @@ public function checkFields(string $user, ?TableNode $body):void {
 		}
 	}
 
+	/**
+	 * @Then the fields of the last response to user :user and space :space should include
+	 *
+	 * @param string $user
+	 * @param string $space
+	 * @param TableNode|null $body
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function checkFieldsOfSpaceSharingResponse(string $user, string $space, ?TableNode $body):void {
+		$this->verifyTableNodeColumnsCount($body, 2);
+	
+		$bodyRows = $body->getRowsHash();
+		$userRelatedFieldNames = [
+			"owner",
+			"user",
+			"uid_owner",
+			"uid_file_owner",
+			"share_with",
+			"displayname_file_owner",
+			"displayname_owner",
+			"additional_info_owner",
+			"additional_info_file_owner",
+			"space_id"
+		];
+
+		$this->setLastPublicShareData($this->getResponseXml(null, __METHOD__));
+		foreach ($bodyRows as $field => $value) {
+			if (\in_array($field, $userRelatedFieldNames)) {
+				$value = $this->substituteInLineCodes($value, $user);
```

Comment:
is this necessary? since the whole substitution is done below with additional function.

Reply:
Yes, I forget delete it